### ISSUE TITLE
[Bug Report][3.5.15] Transitions and animations are not disabled when the user disables them

### ISSUE THREAD

**Issue by thany** [vuetifyjs/vuetify#19622]
### Environment
**Vuetify Version:** 3.5.15
**Vue Version:** 3.4.22
**Browsers:** Firefox 124.0
**OS:** Windows 10

### Steps to reproduce
1. Disable animations in your OS and/or browser
2. Cause any animation or transition to happen. I left the playground contents bog standard, because even something so simple as hovering over the text input causes a transition that ought to be disabled.

### Expected Behavior
No animations when the user chooses to disable them

### Actual Behavior
Animations

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqLrq7VySvNyQETsQApPwV/)

### Other comments
It bares noting that this is an *accessibility issue*. I do not consider this a feature request, for 2 reasons:
1. Users suffering from motion sensitivity (such as yours truly) do not consider having animation to be a missing feature. It causes nauscia and is therefor well and truly a bug.
2. This has never been taken care of, neither in Vetify 3, nor in 2, which frankly baffles me. I would expect a professional UI framework to have this nailed, and yet it seems to have been put to one side for all these years.

**Comment by thany** [comment]
Addendum: In `prefers-reduced-motion` the part "reduced" should not be taken literally. It's an unfortunate name for a term that indicates a requirement for absolutely no motion. Users can suffer from motion sensitivity in varying degrees of intensity, and as an author of a UI framework that implements a11y quite well otherwise, you cannot assume anything but the worst when this "preference" is active.
